test/docker/networktest/Dockerfile

```diff
@@ -0,0 +1,23 @@
+# escape=`
+FROM microsoft/windowsservercore
+
+SHELL ["powershell.exe","-command"]
+
+# the source msi should change on a daily basis
+# the destination should not change
+ADD PSCore.msi /PSCore.msi
+
+# the msipath (PSCore.msi) below will need to change based on the daily package
+# set up for basic auth
+# install-powershellremoting will restart winrm service
+RUN net user testuser 11aa!!AA /add; `
```

Comment:
As long as you are using PowerShell...
New-LocalUser -Name testuser -Password (ConvertTo-SecureString 11aa!!AA -AsPlainText -Force)